source/macos/AdaptiveCards/AdaptiveCards/Renderers/RichTextBlockRenderer.swift

```diff
@@ -0,0 +1,88 @@
+import AdaptiveCards_bridge
+import AppKit
+
+class RichTextBlockRenderer: NSObject, BaseCardElementRendererProtocol {
+    static let shared = RichTextBlockRenderer()
+    
+    func render(element: ACSBaseCardElement, with hostConfig: ACSHostConfig, style: ACSContainerStyle, rootView: NSView, parentView: NSView, inputs: [BaseInputHandler]) -> NSView {
+        guard let richTextBlock = element as? ACSRichTextBlock else {
+            logError("Element is not of type ACSRichTextBlock")
+            return NSView()
+        }
+        
+        let textView = ACRTextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.isEditable = false
+        textView.textContainer?.lineFragmentPadding = 0
+        textView.textContainerInset = .zero
+        textView.layoutManager?.usesFontLeading = false
+        textView.setContentHuggingPriority(.required, for: .vertical)
+        textView.backgroundColor = .clear
+        
+        // init content
+        let content = NSMutableAttributedString()
+        
+        // parsing through the inlines
+        for inline in richTextBlock.getInlines() {
+            let textRun = inline as? ACSTextRun
+
+            if textRun != nil {
```

Comment:
This can be clubbed into one statement using `guard` and indentation can be fixed.
```suggestion
            guard let textRun = inline as? ACSTextRun else { continue logError("// Appropriate message //") }
```

Reply:
fixed